pom.xml

```diff
@@ -31,17 +31,38 @@
     -->
 
     <dependencies>
+
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter-engine</artifactId>
             <version>${junit.jupiter.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
```

Comment:
I liked the fact this was runnable without dependencies, but I suppose with maven:exec it doesn't matter too much.

Reply:
Yeah, I tend to run as a fat JAT with mvn:exec